migrations/9_SCHEME-REP-Auction_Gen.js

```diff
@@ -58,7 +64,7 @@ module.exports = async function (deployer) {
     dxAvatar.address,
     reputationReward,
```

Comment:
this should be rep reward per auction.
so : reputationReward/numberOfGenAuctions

Reply:
Done, thank you!